.github/workflows/release.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   release:
-    uses: hashicorp/web-platform-packages/.github/workflows/release.yml@8f20ba3ccb4ffe9a9bc0b14060af00929e6655f1
+    uses: hashicorp/web-platform-packages/.github/workflows/release.yml@_______FIXME______
```

Comment:
This will be updated with the new SHA when https://github.com/hashicorp/web-platform-packages/pull/218 is merged